src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -118,24 +114,28 @@ var getDateTimeComposition = function (ds){
 
 // handling the datetime format here
 var getDateTime = function (dateTimeString) {
-    if (!validString(dateTimeString))
+    if (!validDatetimeString(dateTimeString))
         return '';
 
-    // handle special datetime format like 20130130080051+0500, 20130130080051-0500
+    // handle the datetime format with time zone
     var ds = dateTimeString.toString();
-    if (/^(\d{14})(-|\+)(\d{4})$/.test(ds))
-    {
-        var dateSections = ds.split(ds[14]);
+    var timeZoneChar = '';
+    if (ds.indexOf('-') !== -1)
+        timeZoneChar = '-';
+    else if (ds.indexOf('+') !== -1)
+        timeZoneChar = '+'; 
+    if (timeZoneChar !== '')
+    {
+        var dateSections = ds.split(timeZoneChar);
         var dateTimeComposition = getDateTimeComposition(dateSections[0]);
         var date = dateTimeComposition.year + '-' + dateTimeComposition.month + '-' + dateTimeComposition.day;
```

Comment:
What if the dateTime is in "YYYY" format? Not sure whether the month and day values are valid.